src/Application/Folders/Commands/AddFolder/AddFolderCommand.cs

```diff
@@ -0,0 +1,73 @@
+using Application.Common.Exceptions;
+using Application.Common.Interfaces;
+using Application.Common.Models.Dtos.Physical;
+using AutoMapper;
+using Domain.Entities.Physical;
+using Domain.Exceptions;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Application.Folders.Commands.AddFolder;
+
+public record AddFolderCommand : IRequest<FolderDto>
+{
+    public string Name { get; init; }
+    public string? Description { get; init; }
+    public int Capacity { get; init; }
+    public Guid LockerId { get; init; }
+}
+
+public class AddFolderCommandHandler : IRequestHandler<AddFolderCommand, FolderDto>
+{
+    private readonly IApplicationDbContext _context;
+    private readonly IMapper _mapper;
+
+    public AddFolderCommandHandler(IApplicationDbContext context, IMapper mapper)
+    {
+        _context = context;
+        _mapper = mapper;
+    }
+
+    public async Task<FolderDto> Handle(AddFolderCommand request, CancellationToken cancellationToken)
+    {
+        var locker = await _context.Lockers
+            .FirstOrDefaultAsync(l => 
```

Comment:
spacing?

Reply:
fixed at https://github.com/ProFile-org/ProFile-BE/pull/80/commits/d87261e746bf0889e0651581bc05148e31e0eef9